src/constants.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const INPUT_MESSAGE = {
+  enterNumber: 'Please enter a number of 4 digits ',
+  enterCorrectNumber: 'Please enter correct number of 4 digits ',
```

Comment:
It is better to indicate that the entered number is incorrect, and not simply to duplicate the previous message with the addition of the word "correct"